src/types.jl

```diff
@@ -303,8 +303,11 @@ eval(
     @convert(
         OGRFieldType::DataType,
         OFTInteger::Int32,
+        OFTInteger::Int16,
+        OFTInteger::Bool,
```

Comment:
To give an example (see my other comment): I suspect the inclusion of these two lines might mangle the conversion of `OFTInteger` to `Int32` because of the `Dict`-based implementation of the `convert` macro.

Reply:
Good point, I missed the reverse one. So because of the `Dict`, the macro has become order dependent.

Reply:
Thanks!